website/docs/reference/global-configs/behavior-changes.md

```diff
@@ -152,3 +154,13 @@ on-run-start:
   - '{{ ... if flags.WHICH != 'freshness' }}'
 ```
 </File>
+
+
+### MetricFlow time spine YAML
+The `require_yaml_configuration_for_mf_time_spines` flag is set to `False` by default.
+
+Set the flag to `True` to require YAML configuration for MetricFlow time spine file for dbt Cloud Versionless or dbt Core 1.9 and later. In previous versions (dbt Core 1.8 and earlier), the MetricFlow time spine configuration was stored in a `metricflow_time_spine.sql` file.
+
+When the flag is set to `True`, dbt will raise a deprecation warning if it detects a MetricFlow time spine configured in a SQL file. When the flag is set to `False`, dbt will continue to support the SQL file configuration. 
```

Comment:
This is the other way around. See my note in the PR description [here](https://github.com/dbt-labs/dbt-core/pull/10857)

> I added this to dbt_project.yaml in jaffle-sl-template and saw no warning for dbt parse:
flags:
  require_yaml_configuration_for_mf_time_spines: True

Reply:
oh right sorry, i missed the tests. thanks for clarifying @DevonFulcher ! so the flag allows users to continue to support the SQL file configuration. I'll update!

Reply:
updated it to this @DevonFulcher :

### MetricFlow time spine YAML
The `require_yaml_configuration_for_mf_time_spines` flag is set to `False` by default.

Set the flag to `True` to require YAML configuration for MetricFlow time spine file for dbt Cloud Versionless or dbt Core 1.9 and later. In previous versions (dbt Core 1.8 and earlier), the MetricFlow time spine configuration was stored in a `metricflow_time_spine.sql` file.

When the flag is set to `True`, dbt will continue to support the SQL file configuration. When the flag is set to `False`, dbt will raise a deprecation warning if it detects a MetricFlow time spine configured in a SQL file. 

The MetricFlow YAML file should be named `models/_models.yml` and should be located in the `models` directory.

Reply:
Looks great! So, both configurations are accepted for now, regardless of the value of `require_yaml_configuration_for_mf_time_spines`, so this line isn't totally accurate:

> Set the flag to True to require YAML configuration for MetricFlow time spine file for dbt Cloud Versionless or dbt Core 1.9 and later.

I think we can just delete that sentence since you describe the True/False behavior later in this section.

Reply:
good flag, removed it now @DevonFulcher !